demos/drawer/drawer.scss

```diff
@@ -14,7 +14,24 @@
 // limitations under the License.
 //
 
+@import "@material/list/mixins";
+@import "@material/form-field/mdc-form-field";
 @import "../common";
 @import "../../packages/mdc-drawer/mdc-drawer";
+@import "../../packages/mdc-drawer/mixins";
 @import "../../packages/mdc-elevation/mdc-elevation";
 @import "../../packages/mdc-list/mdc-list";
+
+.demo-drawer {
+  @include mdc-drawer-fill-color(#3f51b5);
+  @include mdc-drawer-ink-color(#fff);
+  @include mdc-list-item-graphic-ink-color(#fff);
```

Comment:
We should probably call the `primary-text-ink-color` mixin too
(it needs to be called on the `.mdc-list-item` element, btw;
otherwise it will overwrite the value set by `mdc-drawer-fill-color`.)

So we probably want something like:

```scss
.demo-drawer--custom {
  @include mdc-drawer-fill-color($material-color-teal-900);
  @include mdc-drawer-ink-color(#fff);

  .demo-drawer-list-item {
    @include mdc-list-item-graphic-ink-color(text-icon-on-dark);
    @include mdc-list-item-primary-text-ink-color(text-secondary-on-dark);
  }

  .demo-drawer-list-item.mdc-list-item--selected {
    @include mdc-list-item-graphic-ink-color(text-primary-on-dark);
    @include mdc-list-item-primary-text-ink-color(text-primary-on-dark);
  }
}
```

Reply:
ah ok - ya initially I had this in the mixin, but then we took it out. After removing I should have just put it in the demo classes. Makes sense.

Reply:
Should I also do the same changes on `.demo-drawer--accessible`? If so, should I also create mdc-list accessible mixins?